err_fmt_test.go

```diff
@@ -0,0 +1,863 @@
+package clues_test
+
+import (
+	stderr "errors"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/alcionai/clues"
```

Comment:
not sure how you want to handle import ordering in this repo, but if the repo imports are supposed to be on their own this should be in a different block

Reply:
Ah well, we can come back to it.